generator/packets.py

```diff
@@ -109,5 +107,18 @@
         ["remVersion", 4, None, "Version of roboteam_embedded_messages"],
         ["id",         4, None, "Id of the robot"],
         ["message_length", 8, None, "Length of the following message"]
+    ],
+    "BasestationGetConfiguration" : [
+        ["header",     8, None, "Header byte indicating the type of packet"]
+    ],
+    "BasestationConfiguration" : [
+        ["header",     8, None, "Header byte indicating the type of packet"],
+        ["remVersion", 4, None, "Version of roboteam_embedded_messages"],
```

Comment:
remVersion should be 5 now right?

Reply:
4 stands for 4 bits used, not for the version